.github/workflows/tiiuae-pixhawk-artifact-publish.yaml

```diff
@@ -0,0 +1,277 @@
+name: tiiuae-pixhawk
```

Comment:
Please use some other name. This already exists.

Reply:
Name changed